controllers/costmanagement_controller.go

```diff
@@ -132,6 +158,94 @@ func GetClusterID(r *CostManagementReconciler, cost *costmgmtv1alpha1.CostManage
 	return nil
 }
 
+func GetPullSecretToken(r *CostManagementReconciler, costInput *CostManagementInput) error {
+	ctx := context.Background()
+	log := r.Log.WithValues("costmanagement", "GetPullSecretToken")
+	sl := &corev1.SecretList{}
+	err := r.List(
+		ctx,
+		sl,
+		&client.ListOptions{Namespace: openShiftConfigNamespace},
+	)
+	if err != nil {
+		if errors.IsNotFound(err) {
```

Comment:
maybe `swtich-case` will be more readable:

```go
switch {
case errors.IsNotFound(err):
  ....
case errors.IsForbidden(err):
....
default:
....
}
```

Reply:
Done